include/wx/generic/grid.h

```diff
@@ -2619,21 +2619,21 @@ class WXDLLIMPEXP_CORE wxGridEditorCreatedEvent : public wxCommandEvent
         }
 
     wxGridEditorCreatedEvent(int id, wxEventType type, wxObject* obj,
-                             int row, int col, wxControl* ctrl);
+                             int row, int col, wxWindow* ctrl);
 
     int GetRow()                        { return m_row; }
     int GetCol()                        { return m_col; }
-    wxControl* GetControl()             { return m_ctrl; }
+    wxWindow* GetControl()              { return m_ctrl; }
     void SetRow(int row)                { m_row = row; }
     void SetCol(int col)                { m_col = col; }
-    void SetControl(wxControl* ctrl)    { m_ctrl = ctrl; }
+    void SetControl(wxWindow* ctrl)     { m_ctrl = ctrl; }
 
     virtual wxEvent *Clone() const wxOVERRIDE { return new wxGridEditorCreatedEvent(*this); }
 
 private:
     int m_row;
     int m_col;
-    wxControl* m_ctrl;
+    wxWindow* m_ctrl;
```

Comment:
This member is private, so we could also rename it to `m_window` to avoid confusion (the other `m_control` has to remain named like this because it's protected).